db/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.0.2 on 2024-12-11 13:28
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(null=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Race',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(choices=[['Elf', 'Elf'], ['Dwarf', 'Dwarf'], ['Human', 'Human'], ['Ork', 'Ork']], max_length=255, unique=True)),
```

Comment:
The `choices` attribute in the `Race` model should use tuples instead of lists. Change `[['Elf', 'Elf'], ['Dwarf', 'Dwarf'], ['Human', 'Human'], ['Ork', 'Ork']]` to `[('Elf', 'Elf'), ('Dwarf', 'Dwarf'), ('Human', 'Human'), ('Ork', 'Ork')]`.